### ISSUE TITLE
Running the contributed command:'sonarqube-inject.updateBindings' failed.

### ISSUE THREAD

**Issue by calexandre** [silverbulleters/sonarqube-inject-vsc#33]
First time using this extension...
Followed the instructions and ran "SonarQube Inject: Create global config with credentials to servers" and then "SonarQube Inject: Create local sonarlint config with project binding".

When running the 3rd command "SonarQube Inject: Update bindings to SonarQube server" I get an instant error saying that "Running the contributed command:'sonarqube-inject.updateBindings' failed."...

![image](https://user-images.githubusercontent.com/1485193/36673028-557bae14-1af9-11e8-9e95-da51de5ca5b0.png)

**Comment by nixel2007** [comment]
Hello!

Did you check created files and filled the values?

**Comment by calexandre** [comment]
Yes.
Switched to login/password instead of Token and the error vanished.

Still, when I order it to analyze a file or project it does nothing...just sits there without any kind of output.
Is there any way view the output?

**Comment by nixel2007** [comment]
1. sonarlint looks for files in src folder by default. if your files non in src folder, you need to tweak `sonarqube-inject.sourcePath` property in your settings.json
2. sonarlint only analyze file of this languages:

java
javascript
php
python
cobol
abap
plsql
swift
bsl

what is the extension of files you try to analyze?

**Comment by nixel2007** [comment]
> Is there any way view the output?

No, at the moment :( If you see an error (red notification) you can check console at `devevelopment tools` (you can toogle them from command pallete).

**Comment by calexandre** [comment]
That explains it...
I'm trying to analyze C# (dotnet core) code...
All my code is under the src/ folder...

Any plans to support C#? SonarQube already supports it...

**Comment by nixel2007** [comment]
SonarQube use separate sonarlint analyzer for C#. I want to check if I can include C# in list of supported languages, but currently have no time for this (not in this month). :(

**Comment by calexandre** [comment]
Let me know if you need assistance (or a beta tester). I would be delighted to help you out whenever I can..

**Comment by nixel2007** [comment]
Thanks! :)

**Comment by SebGt** [comment]
Hello, 
I tried to enable analyse for PL/SQL code but it doesn't work. 

I setup SonarLint extension and it is working fine for Python files.
I setup SonarQube extension and ran a update bindings to SonarQube server successfully but Analyze current file do nothing.

Any suggest of things to check.  Is there log somewhere to check?
Thanks

**Comment by mayconbeserra** [comment]
Any news about enabling sonarlint analyzer for c#?
